src/BulkIssueCreatorCLI/Program.cs

```diff
@@ -0,0 +1,355 @@
+                    results.Enqueue(current);
+
+                    // if this was a parent for any other issue, those issues can now be considered good to go
+                    foreach (IssueToCreateWithEpic entry in toBeSorted)
+                    {
+                        if (StringComparer.OrdinalIgnoreCase.Equals(entry.EpicTitle, current.Title) &&
+                            StringComparer.OrdinalIgnoreCase.Equals(entry.EpicRepo, current.Repository) &&
+                            StringComparer.OrdinalIgnoreCase.Equals(entry.EpicOrg, current.Organization))
+                        {
+                            entry.HasParent = true;
+                        }
+                    }
+                }
+                else
+                {
+                    toBeSorted.Enqueue(current);
+                }
+            }
+
+            return results.ToList();
+        }
+
+        private static async Task CreateIssuesAsync(List<IssueToCreateWithEpic> parsedIssueData, string progressFile)
+        {
+            // The issues are sorted such that no epic shows up before its parent epic.
+            // this means I should be able to get the parent epic all the time.
+
+            int issuesCreated = parsedIssueData.Count;
+
+            while (parsedIssueData.Count > 0)
+            {
+                // Get the epic at the front
+                IssueToCreateWithEpic issue = parsedIssueData[0];
+
+                Colorizer.WriteLine("Create [Yellow!{0}]? [Green!y]/[Red!n]", issue.Title);
+                if (Console.ReadKey().Key != ConsoleKey.Y)
+                {
+                    // remove the issue from the list
+                    parsedIssueData.Remove(issue);
+                    SaveIssuesToProgressFile(progressFile, parsedIssueData);
+                    continue;
+                }
+
+                using (LoggingScope ls = new LoggingScope("Creating [Yellow!{0}]", issue.Title))
+                {
+                    if (!string.IsNullOrEmpty(issue.EpicTitle))
+                    {
+                        issue.Epic = await GetEpicFromTitle(issue);
+                    }
+
+                    (bool issueCreated, string errorOrUrl) = await s_issueManager.TryCreateNewIssueAsync(issue, false);
+
+                    if (issueCreated == true)
+                    {
+                        Colorizer.WriteLine("Issue [Yellow!{0}] created with url [Cyan!{1}]", issue.Title, errorOrUrl);
+                        parsedIssueData.Remove(issue);
+                        SaveIssuesToProgressFile(progressFile, parsedIssueData);
+                    }
+                    else
+                    {
+                        Colorizer.WriteLine("[Red!Error]: Could not create issue [Yellow!{0}]. [Red!Stopping]", issue.Title);
+                    }
+                    await Task.Delay(500);
+                }
+
+
+            }
```

Comment:
Request: once created, could all the issues be linked together - maybe optionally, if nothing else. Many times I've opened issues in bulk, if I make changes to one I like to update them all but harder to find them all. I need the one ring.

Reply:
They could -- but you could model this in the input file. Or the output of the app would be links to the created issues?

Reply:
If the output contains the links (which I think it should anyway - the `gh` CLI does) I still have to update all the bugs or find somewhere to store those links. I'd rather them be in the bug so all the related bugs (epics, etc.) are linked and I can flip between then to update them easily, as I often try to do with epics I've opened against all languages.

Reply:
I wil create a feature ask for this.

Reply:
#42